cra-rxjs-styled-components/.env.production

```diff
@@ -0,0 +1,2 @@
+REACT_APP_API_URL=https://api.starter.dev/api
```

Comment:
I'd rather keep a single `.env` file like the original was attempting and then use repo secrets to recreate. The secrets can get stored in Amplify and the build should pick it up and replace the values.

Reply:
@dustinsgoodman I've reverted the changes and added both environmental variables to Amplify. The branch will need to be merged with the new `amplify.yml` file in order to reflect the new site on AWS.